timemachine/cpp/src/barostat.hpp

```diff
@@ -26,10 +26,10 @@ template <typename RealType> class MonteCarloBarostat {
     ~MonteCarloBarostat();
 
     // inplace_move() may modify d_x and d_box
-    void inplace_move(double *d_x, double *d_box, cudaStream_t stream);
+    void inplace_move(CoordsType *d_x, CoordsType *d_box, cudaStream_t stream);
 
     // used for testing, bool return tells you if move was accepted
-    bool inplace_move_host(double *h_x, double *h_box);
+    bool inplace_move_host(double *h_x, CoordsType *h_box);
```

Comment:
CoordsType h_x?

Reply:
Good catch, fixed in https://github.com/proteneer/timemachine/pull/1172/commits/3eb4dafe59255ad9cceeddd4d88dc231ad611c85